package.json

```diff
@@ -10,11 +10,11 @@
     "format": "prettier .",
     "format:write": "prettier --write .",
     "format:check": "prettier --check .",
-    "test": "node --test",
+    "test": "node --test --experimental-test-module-mocks",
     "test:coverage": "c8 npm test",
-    "test:ci": "c8 --reporter=lcov node --test --test-reporter=@reporters/github --test-reporter-destination=stdout --test-reporter=junit --test-reporter-destination=junit.xml --test-reporter=spec --test-reporter-destination=stdout",
-    "test:update-snapshots": "node --test --test-update-snapshots",
-    "test:watch": "node --test --watch",
+    "test:ci": "c8 --reporter=lcov node --test --experimental-test-module-mocks --test-reporter=@reporters/github --test-reporter-destination=stdout --test-reporter=junit --test-reporter-destination=junit.xml --test-reporter=spec --test-reporter-destination=stdout",
+    "test:update-snapshots": "node --test --experimental-test-module-mocks --test-update-snapshots",
+    "test:watch": "node --run test --watch",
```

Comment:
```suggestion
    "test:watch": "node --run test -- --watch",
```

Reply:
Otherwise you're running --watch against the node --run command which will give an error asking for --watch-path

Reply:
Ahh, my intent was to pass it as an argument to run, got it, I'll adjust accordingly